src/components/snapshots/takeSnapshot.ts

```diff
@@ -175,7 +175,7 @@ const copyFiles = async (
   filePaths.push(...fileRecords.map((fileRecord) => fileRecord.thumbnailPath))
   const baseFilePaths = options.resetFiles ? await getBaseFiles(FILES_FOLDER) : []
 
-  for (const filePath of [...filePaths, ...baseFilePaths]) {
+  for (const filePath of [...filePaths]) {
```

Comment:
You'll need to remove this from `useSnapshot` too, and then remove the `getBasePaths` function and all references to it.

I'll want to test it again after that just to be sure, but I think it'll be okay